### PR TITLE
bug fixed

### PR DESCRIPTION
**Pull request by MuthuKumari-P**

### CODE REVIEW

File: src/main/java/in/muthukumari/controller/BankDetailsController.java [hunk 1]
```diff
@@ -38,18 +29,13 @@ public BankDetailsController() {
...
-	 * @throws DBException 
 	 */
 	public String getIfscCode(String bankName, String branchName)
-			throws InvalidException {
-		
+			 {
 		String ifscCode = null;
 		try {
 			ifscCode = BankDetailService.getIfscCode(bankName, branchName);
-		} catch (ClassNotFoundException | SQLException e) {
```
Comment (selva08sankari) [inline src/main/java/in/muthukumari/controller/BankDetailsController.java:52]: add e.printstacktrace

File: src/main/java/in/muthukumari/controller/BankDetailsController.java [hunk 2]
```diff
@@ -1,35 +1,26 @@
...
-	}
-	
 	/**
 	 * This method used to get the branch name list of each bank
 	 * @param bankName
 	 * @return
-	 * @throws ClassNotFoundException
-	 * @throws SQLException
 	 */
-	public static Set<String> getBranchNameList(String bankName) throws ClassNotFoundException, SQLException {
```
Comment (selva08sankari) [inline src/main/java/in/muthukumari/controller/BankDetailsController.java:28]: handle ClassNotFoundException, SQLException  in dao itself

**Review by selva08sankari — CHANGES_REQUESTED** [review]

File: src/main/java/in/muthukumari/controller/BankDetailsController.java (hunk 1 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/controller/BankDetailsController.java:52]: e.printstacktrace shows the issue in sonar cloud. so I didn't use this. But I use logger method

File: src/main/java/in/muthukumari/controller/BankDetailsController.java (hunk 2 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/controller/BankDetailsController.java:28]: Already I did changed this.

**Pull request merged**